### DISCUSSION TITLE
Feature request: Latency + duration logs for each step

### DISCUSSION THREAD

**Discussion by satvikreddy** [voxos-ai/bolna#234] [Ideas]
Telephony in => STT => LLM => TTS => Telephony out

If there was a log for every iteration of the above, it'll be very helpful.
This will help in detecting bottlenecks and compare Configs when switching out individual components.

It might be complex as there are streams everywhere, but some basic version to get a sense of where the delays are will be great.

**Comment by marmikcfc** [comment]
Hey @satvikreddy it's already available. After running the code you should be seeing a `logs` directory. Under the logs directory, you will have agents and under them csv file with unix timestamp. Those are the same logs.

Let me know if those files aren't being generated.

**Reply by rrajzenon** [reply]
I cant see any logs directory. Is there any parameter that I should change?
